client/src/components/displayPictures.js

```diff
@@ -1,43 +1,84 @@
-import React, {Component} from 'react';
-import RenderProfilePage from './profile';
+import React, { Component } from "react";
 import axios from "axios";
-
+import {
+  Container,
+  Row,
+  Col,
+  Card,
+  Button,
+  InputGroup,
+  FormControl
+} from "react-bootstrap";
 
 class RenderProfilePictures extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      img: []
+    };
+  }
 
-    constructor(props) {
-        super(props);
-        this.state = {
-        img: null
-        };
-    }
-
-    componentDidMount() {
-        axios
-          .get("http://localhost:8000/files")
-          .then(res => {
-            this.setState({
-              img: res.data
-            });
+  componentDidMount() {
+    axios
+      .get("http://localhost:8000/files", {
+        headers: { token: sessionStorage.getItem("jwt") }
+      })
+      .then(res => {
+        //Fill an array with the files sent back from the server. Each array element will use the filename and description
+        //of the current element.
+        var arr = [];
+        for (var i = 0; i < res.data.length; i++) {
```

Comment:
Glad you figured out how to store all the file names to later load the images, I was having a tough time approcahing this.